# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mc/0004-mc-4.1.40-pre9-ext2_ino_t.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: The type of the 2nd argument of ext2fs_get_next_inode() needs to be 'ext2_ino_t *'

Reference: /usr/include/ext2fs/ext2fs.h

--- mc-4.1.40-pre9/vfs/undelfs.c.orig	2011-04-12 17:15:30.912076572 +0200
+++ mc-4.1.40-pre9/vfs/undelfs.c	2011-04-12 17:15:51.760071243 +0200
@@ -163,7 +163,7 @@
 undelfs_loaddel ()
 {
     int retval, count;
-    ino_t ino;
+    ext2_ino_t ino;
     struct ext2_inode inode;
     ext2_inode_scan   scan;
     
